config/config.go

```diff
@@ -717,6 +717,8 @@ type Performance struct {
 	MemoryUsageAlarmRatio float64 `toml:"memory-usage-alarm-ratio" json:"memory-usage-alarm-ratio"`
 
 	EnableLoadFMSketch bool `toml:"enable-load-fmsketch" json:"enable-load-fmsketch"`
+
+	LiteInitStats bool `toml:"lite-init-stats" json:"lite-init-stats"`
```

Comment:
Can we use a SQL variable instead of a config item here? SQL variables are more flexible and east-to-use.

Reply:
1. `LiteInitStats` only takes effect when one tidb-server starts. If you change its value but no tidb-server restarts, nothing takes effect. We don't need the flexibility of changing its value online.
2. Making `LiteInitStats` a configuration can provide instance-level config. For example, you can make one tidb-server's `lite-init-stats` true and the other false to observe their different behaviors, which is more elegant than using a global variable to archive that.